Code-Hammers/code-hammers
# [CHE-120] Add Total Posts Display On Threads

### Description
This PR aims to:

- add total post count on individual threads
- add total reply count per thread on threadDisplay for All
- add total reply count per thread on threadDisplay custom forums 

Extra:  
- add to queryHelper in `server/controllers/helpers/queryHelpers.ts` to handle MongoDb aggregate for sorting
- add type to forums for postCount

### Jira Task

[CHE-120](https://codehammers.atlassian.net/jira/software/projects/CHE/list?selectedIssue=CHE-120&text=CHE-120)

### Testing Instructions
- Ensure branches are up to date.
- Navigate to Forums and click through All Forums and each Forum (ex: General) to see if reply count appears on the screen for each individual thread.
- Click into the individual threads, and there should be a total post count for that thread (including the OP)

### Checklist

#### All Team Members


### Additional Notes, Images, etc.

(This is space for any additional information or attachments you'd like to add as part of this PR.)


[CHE-120]: https://codehammers.atlassian.net/browse/CHE-120?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
